### PR TITLE
Documentation: top-level redirect for github pages

### PR DESCRIPTION
**Pull request by evan-masseau**
Adds a top-level redirect so that the root page of GH pages doesn't 404 and always points to latest docs (we just have to update this html file in the version bump commit)

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<!-- Redirect to latest version -->
+<meta HTTP-EQUIV="REFRESH" content="0; url=./1.4.1/index.html">
```
Comment (evan-masseau) [inline docs/index.html:2]: So we'd just update this version instead of the link in the readme in the future.

Comment (kennyklaviyo) [reply docs/index.html:2]: should we call this out in the comment where we update version to do it here instead of the readme?

Comment (evan-masseau) [reply docs/index.html:2]: yes

Comment (jordan-griffin) [reply docs/index.html:2]: Probably want to make sure the process doc on Android releases is up to date as well.

**Review by jordan-griffin — APPROVED** [review]

**Review by kennyklaviyo — APPROVED** [review]

File: docs/index.html (hunk 1 above)
Comment (evan-masseau) [reply docs/index.html:2]: I Will keep wiki synced

**Pull request merged**